source/dub/package_.d

```diff
@@ -721,12 +722,16 @@ class Package {
 
 		// check for default source folders
 		immutable hasSP = ("" in bs.sourcePaths) !is null;
+		immutable hasCSP = ("" in bs.cSourcePaths) !is null;
 		immutable hasIP = ("" in bs.importPaths) !is null;
+		immutable hasCIP = ("" in bs.cImportPaths) !is null;
 		if (!hasSP || !hasIP) {
 			foreach (defsf; ["source/", "src/"]) {
 				if (existsFile(m_path ~ defsf)) {
 					if (!hasSP) bs.sourcePaths[""] ~= defsf;
+					if (!hasCSP) bs.cSourcePaths[""] ~= defsf;
 					if (!hasIP) bs.importPaths[""] ~= defsf;
+					if (!hasCIP) bs.cImportPaths[""] ~= defsf;
```

Comment:
don't auto initialize cSourcePaths and cImportPaths to `source/`

Reply:
Removed.